src/test/functional/utlis/puppeteer.util.ts

```diff
@@ -19,15 +21,15 @@ export const goTo = async (url: string) => {
 };
 
 export const getPageTitle = async () => {
-  return await scope.page.title();
+  return scope.page.title();
```

Comment:
i think the await is quite important in puppeteer calls. not sure if it needed here tho.

Reply:
The `await` here is [unnecessary](https://eslint.org/docs/rules/no-return-await) since the whole function is an `async` method, so the caller will need to await it again: `await getPageTitle()`.